docs/webhooks.md

```diff
@@ -0,0 +1,148 @@
+## Implémentation d'un wehbook
+
+Datapass possède un système de webhooks permettant de souscrire aux différents
+changement d'état d'une demande. Par défaut ce système est désactivé (en faveur
+d'emails de notifications et d'un appel via un bridge (code spécifique au
+fournisseur de service) lors d'une validation d'une demande).
+
+Si vous voulez avoir un contrôle plus fin du cycle de vie des demandes, vous
+pouvez utiliser ce système (par exemple pour gérer un CRM, mettre en place des
+statistiques..).
+
+Le système de webhooks utilise l'approche du token de vérification et du header
+`X-Hub-Signature` permettant d'authentifier les appels depuis le endpoint cible.
+
+L'implémentation des webhooks s'effectue en 2 étapes: une partie sur Datapass et
+une partie sur votre système.
+
+### Partie 1: Datapass
```

Comment:
Datapass => DataPass

I use this case everywhere else. I'll be delighted if we could keep consistency on this.